### PR TITLE
Feat/sgenetwork testnet update

### PR DESCRIPTION
**Pull request by shang-chi-7**

### CODE REVIEW

File: testnets/sgetestnet/chain.json [hunk 1]
```diff
@@ -75,8 +75,8 @@
     ]
   },
   "logo_URIs": {
-    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sgenetworktestnet/images/logo.png",
-    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sgenetworktestnet/images/logo.svg"
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sgetestnet/images/logo.png",
```
Comment (JeremyParish69) [inline testnets/sgetestnet/chain.json:78]: these images don't exist (by that name)

File: testnets/sgetestnet/assetlist.json [hunk 2]
```diff
@@ -17,11 +17,11 @@
         }
       ],
       "base": "usge",
-      "name": "SGE Network",
+      "name": "SGE Testnet",
       "display": "sge",
       "symbol": "SGE",
       "logo_URIs": {
-        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sgenetworktestnet/images/sge-logo.png"
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sgetestnet/images/sge-logo.png"
```
[the remark this answers is not in the document]
Comment (shang-chi-7) [reply testnets/sgetestnet/assetlist.json:24]: @JeremyParish69 What will be the correct path ?

[the remark this answers is not in the document]
Comment (shang-chi-7) [reply testnets/sgetestnet/assetlist.json:24]: https://raw.githubusercontent.com/cosmos/chain-registry/master/sgetestnet/images/sge.png

**Comment by reeves89** [comment]
@JeremyParish69 don't merge it today, we might need to update the ibc details, due to the long inactivity client is frozen.

**Comment by shang-chi-7** [comment]
@JeremyParish69 Please review

**Pull request merged**